proto/da/da.proto

```diff
@@ -79,6 +79,7 @@ message GetIdsRequest {
 // GetIdsResponse is the response type for the GetIds rpc method.
 message GetIdsResponse {
 	repeated ID ids = 1;
+  int64 timestamp = 2;
```

Comment:
Did u look at google.protobuf.Timestamp instead of int64 for better precision and proto conversions?

Reply:
I did, but I got some issues with buf build setup.
I'll give it another try.

Reply:
Yes, i also had issue when doing go-sequencing. I used cosmos gogoproto. See https://github.com/rollkit/go-sequencing/blob/main/go.mod#L6

Reply:
Still can't get working CI....